docs/.linkcheckerrc

```diff
@@ -3,12 +3,13 @@ maxrequestspersecond=1
 recursionlevel=1
 anchors=1
 sslverify=0
+threads=1
 
 [filtering]
 ignore=
   pythonapi/lightgbm\..*\.html.*
   http.*amd.com/.*
-ignorewarnings=http-robots-denied,https-certificate-error
+ignorewarnings=http-robots-denied,https-certificate-error,url-rate-limited
```

Comment:
I'm afraid that this change can hide real problems. With ignoring `url-rate-limited` link can either be fine or be broken. And this warning can be avoided by manual re-run (I asked to provide such possibility via `dispatch` trigger).

Reply:
```suggestion
ignorewarnings=http-robots-denied,https-certificate-error
```
ok